### PR TITLE
Fix issue2309 1

### PR DESCRIPTION
**Pull request by hasa1K**
https://github.com/actiontech/sqle/issues/2309

### CODE REVIEW

File: internal/dms/biz/access_token.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+type AuthAccessTokenUsecase struct {
+	userUsecase *UserUsecase
+	log         *utilLog.Helper
+}
+
+func NewAuthAccessTokenUsecase(log utilLog.Logger, usecase *UserUsecase) *AuthAccessTokenUsecase {
+	au := &AuthAccessTokenUsecase{
+		userUsecase: usecase,
+		log:         utilLog.NewHelper(log, utilLog.WithMessageKey("biz.license")),
```
Comment (LordofAvernus) [inline internal/dms/biz/access_token.go:24]: 改一下log key

File: internal/dms/biz/access_token.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			// 如果不存在JWTLoginType字段，代表是账号密码登录获取的token或者是扫描任务的凭证，不进行校验
+			loginType, ok := claims[jwtPkg.JWTLoginType]
+			if !ok {
+				return next(c)
+			}
+			if loginType != AccessTokenLogin {
+				return echo.NewHTTPError(http.StatusUnauthorized, "access token login type is error")
+			}
+			uidStr := fmt.Sprintf("%v", claims[jwtPkg.JWTUserId])
+			accessTokenInfo, err := au.userUsecase.repo.GetAccessTokenByUser(context.TODO(), uidStr)
```
Comment (LordofAvernus) [inline internal/dms/biz/access_token.go:56]: c.Request.Contest 代替contest.TODO

File: internal/dms/biz/access_token.go (hunk 1 above)
Comment (hasa1K) [reply internal/dms/biz/access_token.go]: resolve

File: internal/dms/biz/access_token.go (hunk 2 above)
Comment (hasa1K) [reply internal/dms/biz/access_token.go]: 已解决

File: pkg/dms-common/api/jwt/jwt.go [hunk 3]
```diff
@@ -31,6 +32,19 @@ func GenJwtToken(customClaims ...CustomClaimFunc) (tokenStr string, err error) {
 		JWTExpiredTime: jwt.NewNumericDate(time.Now().Add(24 * time.Hour)),
 	}
 
+	return genJwtToken(mapClaims, customClaims...)
+}
+
+func GenJwtTokenWithExpirationTime(expiredTime *jwt.NumericDate, customClaims ...CustomClaimFunc) (tokenStr string, err error) {
+	var mapClaims = jwt.MapClaims{
+		"iss":          "actiontech dms",
```
Comment (LordofAvernus) [inline pkg/dms-common/api/jwt/jwt.go:40]: 这个iss是啥

**Review by LordofAvernus — APPROVED** [review]

**Pull request merged**